dotnet/Capstone/Controllers/BirdSightingController.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Capstone.DAO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System;
+using Capstone.Models;
+using Capstone.Exceptions;
+using System.Security.Principal;
+
+namespace Capstone.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BirdSightingController : ControllerBase
+    {
+        private readonly IBirdSightingDao birdSightingDao;
+        public BirdSightingController(IBirdSightingDao birdSightingDao)
+        {
+            this.birdSightingDao = birdSightingDao;
+
+        }
+
+        [HttpGet("/birds/{birdId}/notes")]
+        public List<BirdSighting> ListAllBirds(int birdId)
+        {
+            List<BirdSighting> sightingList = birdSightingDao.getSightings(birdId);
+
+            if (sightingList == null)
+            {
+                Console.WriteLine("No bird");
+            }
+
+            return sightingList;
+
+        }
+
+        [HttpPost("/newNote")]
```

Comment:
The object sent back with this method does not have the correct ID. Since the get method sends back objects with different IDs, I assume you're sending back the object passed as a parameter without updating the ID.

Reply:
Yeah, I saw this too. Normally when I do a create method, I do it as a void option. However with .NET to be able to get the created method, I had to make it return something.  Even though the immediate return object has an id of 0, it does create correct ids. if you select all Notes, you should be able to see the created Note with the correct id.

Reply:
Looks like you can just pass a null value in place of the `newBirdSighting` variable if you'd like to return a "void". Alternately, you can use the OUTPUT keyword in your SQL statement (you can reference line 82 of `UserProfileSqlDao` for an example, though it is an UPDATE statement).

Either is fine, but I'd rather see the return value be accurate or non-existant, personally. It does look like the frontend doesn't use any return value, so the null option is probably easier.